notebooks/README.md

```diff
@@ -1,9 +1,11 @@
 # README
 
-1. Make sure that you have Octave installed. 
+1. Make sure that you have Octave installed.
 2. If you have Conda/Jupyter/pip installed, go to step 4.
+Check is Conda installed properly by `conda list` typing into your terminal
```

Comment:
btw, I have the 'Anaconda navigator app' installed but if I tyoe `conda list` the terminal says tha does not know the conda command

Reply:
ok somehow re installing Anaconda worked